index.html

```diff
@@ -267,6 +267,24 @@ <h4>Skills</h4>
             </div>
             
         </section>
+        <section class="contact">
+            <div class="contact_me">
+                <h2>Contact me</h2>
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            </div>
+            
+            <form  action="https://formspree.io/f/xdovkznb" method="POST">
+                <input class="form_details" type="text" name="Name" placeholder="Full Name" maxlength="30" required>
+                <input class="form_details" type="email" name="Email" placeholder="Email" required>
+                <textarea class="form_details_message" name="Message" id="" cols="30" rows="10" maxlength="500" placeholder="Write Your Message Here"></textarea>
+
```

Comment:
Please consider adding the `required attribute` to the text area field also. This will help in the validation when submitting the form.